ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -253,3 +253,33 @@ ${local.skip_on_weekends}
     action_group = var.action_group_ids
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "bulk_results_upload" {
+  name                = "${var.env}-bulk_results_upload"
+  description         = "${local.env_title} alert when bulk uploads fail"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+
+  action {
+    action_group = var.action_group_ids
+  }
+
+  data_source_id = var.app_insights_id
+  enabled        = contains(var.disabled_alerts, "bulk_results_upload") ? false : true
+
+  query = <<-QUERY
+requests
+| where true
+| where timestamp >= ago(5m)
+    and name == "/upload/results"
+    and toint(resultCode) >= 500
+  QUERY
+
+  severity    = 1
+  frequency   = 5
+  time_window = 5
```

Comment:
nit: if `frequency` = `time_window` and the timestamp filter, is there a small chance something could slip through the cracks if there's any slight variation in fire time?
I noticed for the batched uploader failures we have a window of 11 min - not sure if that's why

Reply:
That seems like a thing that could happen.